### ISSUE TITLE
Failing doc-test

### ISSUE THREAD

**Issue by tgsmith61591** [charlesdrotar/alkaline#8]
Figure this one out, Taylor...

**Comment by charlesdrotar** [comment]
This was due to an additional ``...`` line within the doctest. This has been addressed within [pull#7](https://github.com/charlesdrotar/alkaline/pull/7).

**Comment by charlesdrotar** [comment]
This is caused by the final line in the doctest.

```python
    Examples
    --------
    >>> @since('0.1')
    >>> def some_fun():
    ...     '''Some docstring'''
    ...     return None
    ...
    >>> some_fun.__doc__
    'Some docstring\n\n.. versionadded:: 0.1' # THIS LINE IS NOT PARSABLE AND CAUSES ERRORS.
```

**Comment by tgsmith61591** [comment]
Fixed in #7
